yt_dlp/extractor/ard.py

```diff
@@ -360,11 +361,30 @@ def _extract_episode_info(self, title):
     def _real_extract(self, url):
         display_id = self._match_id(url)
 
+        query = {
+            'embedded': 'false',
+            'mcV6': 'true'
+        }
+        headers = {}
+
+        ams_cookie = self._get_cookies("https://sso.ardmediathek.de/sso/token").get("ams")
+        if ams_cookie:
+            token = self._download_json('https://sso.ardmediathek.de/sso/token', display_id,
+                                        fatal=False, note='Getting token for age verification')
```

Comment:
let's use a class variable for `https://sso.ardmediathek.de/sso/token`, e.g.
```py
    _TOKEN_URL = 'https://sso.ardmediathek.de/sso/token'
```
and then
```suggestion
        if self._get_cookies(self._TOKEN_URL).get('ams'):
            token = self._download_json(
                self._TOKEN_URL, display_id, 'Fetching token for age verification',
                'Unable to fetch age verification token', fatal=False)
```